parcels/particleset.py

```diff
@@ -238,38 +254,113 @@ def execute(self, pyfunc=AdvectionRK4, starttime=None, endtime=None, dt=1.,
             self.remove(to_remove)
 
     def show(self, **kwargs):
-        if plt is None:
-            raise RuntimeError("Visualisation not possible: matplotlib not found!")
-
-        field = kwargs.get('field', True)
-        lon = [p.lon for p in self]
-        lat = [p.lat for p in self]
+        savefile = kwargs.get('savefile', None)
+        vector = kwargs.get('vector', False)
```

Comment:
My original idea was for vectorplotting to be initiated by calling `field='vector'`. In this implementation, `vector` and `field` are separate arguments. 
Not sure what's best. The advantage of the `field='vector'` notation is that there can never be confusion about what to plot if both `field` _and_ `vector` are provided as arguments (currently, only only vectors are plotted in that case).